certbot-apache/certbot_apache/interfaces.py

```diff
@@ -272,6 +239,24 @@ class BlockNode(ParserNode):
 
     The applicable parameters are dependent on the underlying configuration language
     and its grammar.
+
+    BlockNode objects should have the following attributes:
+
+    # True if this BlockNode is enabled and False if it is inside of an
+    # inactive conditional block. If a BlockNode contains an unmatched
+    # conditional statement, it should itself be flagged as enabled as it's
+    # parsed, but its children should be flagged as disabled.
+    enabled: bool
+
+    # Name, or key of the configuration directive
+    name: str
+
+    # Tuple of parameters of this ParserNode object, excluding whitespaces.
+    parameters: Tuple[str]
```

Comment:
```suggestion
    parameters: Tuple[str, ...]
```

Reply:
Oh, right, as it'll be empty if no parameters are defined.